accounts/scwallet/README.md

```diff
@@ -0,0 +1,60 @@
+# Using the smartcard wallet
+
+## Requirements
+
+  * A USB smartcard reader
+  * A keycard that supports the status app
+
+## Preparing the smartcard
+
+  You can use status' [keycard-cli](https://github.com/status-im/keycard-cli) and you should get version 2.1.1 of their [smartcard application](https://github.com/status-im/status-keycard/releases/download/2.1.1/keycard_v2.1.1.cap)
+
+  You also need to make sure that the PCSC daemon is running on your system.
+
+  Then, you can install the application to the card by typing:
+
+  ```
+  keycard install -a keycard_v2.1.cap
+  ```
+
+  Then you can initialize the application by typing:
+
+  ```
+  keycard init
+  ```
+
+  Finally, you need to have the card generate a new master key:
+
+  ```
+  keycard shell <<END
+  keycard-select
+  keycard-set-pairing PAIRING_KEY PAIRING_INDEX
```

Comment:
does the use need to pair the keycard using `geth` between the `init` and the `shell` steps?
Do I need to get the `PAIRING_KEY` from the `smartcards.json` file or do we show it somewhere?

Reply:
thanks for pointing that out, I have now fixed the instructions